src/crud.js

```diff
@@ -33,32 +33,35 @@ models: {
 
 export default (server, model, { prefix, defaultConfig: config, models: permissions }) => {
   const modelName = model._singular;
+  const permissionsIsArray = Array.isArray(permissions);
```

Comment:
Sure. I'd prefer just `if (Array.isArray(permissions))` though, so that permissions can be `null` or `false`, that gives the user the flexibility to programmatically generate the config and explicitly mark permissions as "off". Thoughts?